uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/src/leave-absences/absence-tab/components/absence-tab-work-patterns.js

```diff
@@ -145,6 +157,10 @@ define([
 
         return workPattern;
       });
+
+      vm.customWorkpattern = _.sortBy(vm.customWorkpattern, function (customWorkpattern) {
```

Comment:
This syntax does not work with new version of underscore. So used `_.chain()` which is supported in both versions of underscore.
Also changed to `vm.customWorkPatterns`.